lib/enkf/local_dataset.cpp

```diff
@@ -135,3 +135,36 @@ int local_dataset_get_size( const local_dataset_type * dataset ) {
 hash_iter_type * local_dataset_alloc_iter(const local_dataset_type * dataset) {
   return hash_iter_alloc(dataset->nodes);
 }
+
+bool local_dataset_has_row_scaling(const local_dataset_type * dataset, const char * key) {
+  return false;
+}
+
+stringlist_type * local_dataset_alloc_scaled_keys(const local_dataset_type * dataset) {
+  stringlist_type * keys = stringlist_alloc_new();
+  hash_iter_type *node_iter = hash_iter_alloc( dataset->nodes );
+
+  while (!hash_iter_is_complete( node_iter )) {
+    const char * key = hash_iter_get_next_key( node_iter );
+    if (local_dataset_has_row_scaling(dataset, key))
+      stringlist_append_copy( keys, key);
```

Comment:
`keys.push_back(key)`

Reply:
I have used `emplace_back( )` to construct from `const char* `